index.html

```diff
@@ -40,6 +42,29 @@ <h3>I’am a Software Developer</h3>
                 <li><a href="#"><i class="fab fa-medium-m"></i></a></li>
             </ul>
         </section>
-    </main>
+        <section class="graphics">
```

Comment:
This section is just an extra piece of code. Please wrap all the content inside one section which is the headline section.

Reply:
Thank you for your quick reply. 
This quote message is from the previous reviewer: "The whole Headline should be wrapped inside a section container. You may use section tags for each section inside a section container."

I am so sorry if I am asking too many questions. I just want to do it right.

Reply:
Hi,
All issue has been resolved.
Thank you for your support.